core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -234,7 +234,7 @@ long getNewBlockIdForFile(AlluxioURI path) throws FileDoesNotExistException, Inv
   /**
    * @return a copy of the current mount table
    */
-  Map<String, MountPointInfo>  getMountTable();
+  Map<String, MountPointInfo>  getMountInfo();
```

Comment:
Why is this renamed? mount info sounds like it is the info for a single mount, but mount table sounds like the entire mapping?

Reply:
I renamed it because I added another method which actually returns the `MountTable` object. I felt it was more apt to call that `getMountTable`. This was used by a few other places and I couldn't think of a good name. Since it wasn't the actual `MountTable` object I just settled on `getMountInfo`. I'm open to other ideas